Libraries/Text/RCTTextAttributes.h

```diff
@@ -78,6 +81,11 @@ extern NSString *const RCTTextAttributesTagAttributeName;
 - (UIColor *)effectiveForegroundColor;
 - (UIColor *)effectiveBackgroundColor;
 
+/**
+ * Text transformed per 'none', 'uppercase', 'lowercase', 'capitalize'
+ */
+- (NSString*) effectiveText: (NSString*) text;
```

Comment:
Nit: Fix spaces here to match the project code style `- (NSString *)effectiveText:(NSString *)text;`